### PR TITLE
Add import / export toggle.

### PR DESCRIPTION
**Pull request by christh**
* and spruce things up a bit.

### CODE REVIEW

File: src/js/toggleImportExport.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+function importExportToggle() {
+	document.getElementById("Automationtemplates").click();
```
Comment (ewerkman) [inline src/js/toggleImportExport.js]: Can you add a `?` after `getElementById("Automationtemplates")` to check for null value? So it should look like this:

`document.getElementById("Automationtemplates")?.click();`

On my environment I didn't have automation templates, so it didn't work (better check all of them)

**Review by ewerkman — CHANGES_REQUESTED** [review]
Hi @christh can you check my comment?

File: src/js/toggleImportExport.js (hunk 1 above)
Comment (christh) [reply src/js/toggleImportExport.js]: Ah, was running it under an Admin account (don't tell). Thanks for that, will get this sorted...

Comment (christh) [reply src/js/toggleImportExport.js]: > Hi @christh can you check my comment?
@ewerkman  Have made the code more defensive - please see updated PR, thanks!

**Pull request merged**